examples/tabbed-document-ui/src/widgets/tab_bar.rs

```diff
@@ -106,22 +107,13 @@ struct TabBarWidget {
 
 impl MakeWidget for TabBarWidget {
     fn make_widget(self) -> WidgetInstance {
-
         let tab_bar = [
-            Stack::new(Orientation::Column, self.tab_items)
-                .make_widget(),
-            Expand::empty()
-                // FIXME this causes the tab bar to take the entire height of the area under the toolbar unless a height is specified
-                //       but we don't want to specify a height in pixels, we want the height to be be automatic
-                //       like it is when the background color is not specified.
-                .with(&WidgetBackground, VERY_DARK_GREY)
-                // FIXME remove this, see above.
-                .height(Px::new(38))
-                .make_widget(),
+            Stack::new(Orientation::Column, self.tab_items).make_widget(),
+            Expand::empty().make_widget(),
```

Comment:
The `Expand` widget is inheriting the `VERY_DARK_GREY` background from line 115, so there's no need to set the background color here again. If you did want a different background color, `Expand::empty()` is actually just [`Expand::new(Space::empty())`](https://github.com/khonsulabs/cushy/blob/f08b7064ec5effb93eb92bea5a3e0f9e71f13544/src/widgets/expand.rs#L63). The way to give this area a specific color would be to color the `Space` widget, and expand it. `Space::colored` produces an area with a given color, or `Space::empty().with(&WidgetBackground...)` would also work.